src/components/Tables/TableFilterAndExtraColumnsBar.tsx

```diff
@@ -64,12 +63,36 @@ export const filtersToComponents = <Data,>(
     });
 };
 
-const TableFilterAndExtraColumnsBar = <Data,>(props: Props<Data>): React.ReactElement => {
+const TableFilterAndExtraColumnsBar = <Data, Filter extends CoreFilter>(
+    props: Props<Data, Filter>
+): React.ReactElement => {
+    const handleClear = (): void => {
+        props.setFilters &&
+            props.filters &&
+            props.setFilters(
+                props.filters?.map((filter) => ({
+                    ...filter,
+                    state: filter.initialState,
+                }))
+            );
+        props.setAdditionalFilters &&
+            props.additionalFilters &&
+            props.setAdditionalFilters(
+                props.additionalFilters?.map((filter) => ({
+                    ...filter,
+                    state: filter.initialState,
```

Comment:
we need extends CoreFilter because we need to know that the filter has a state and an initialState